doc/main.html

```diff
@@ -1235,6 +1249,272 @@ <h3>Git commit hooks</h3>
 
   </section>
 
+
+  <section class="annex" id="sec-document-status-workflow">
```

Comment:
This annex should be moved last to avoid needlessly renumbering existing annexes.

Reply:
Addressed in https://github.com/SMPTE/html-pub/pull/139/commits/6ab6d4dd994fa0751b01ac792613709907ee3b96